Tests/TensorFlowTests/LazyTensorOperationTests.swift

```diff
@@ -33,17 +33,17 @@ final class LazyTensorOperationTests: XCTestCase {
         let op0 = LazyTensorOperation(
             _id: "0", name: "Identity", outputCount: 1)
         op0.addInput(zeroTFEHandle)
-        XCTAssertEqual("\(op0)", "Identity_0[](conc):1")
+        XCTAssertEqual("\(op0)", "Identity_0[](0.0):1")
```

Comment:
```suggestion
        XCTAssertEqual(op0.description, "Identity_0[](0.0):1")
```

Usually, instead of using string interpolation for things that do not need interpolation `"\(x)"` we'd prefer calling the `x.description` property directly or using `String(reflecting: x)` when the type does not conform to `CustomStringConvertible`.